main.py

```diff
@@ -0,0 +1,68 @@
+﻿from flask import Flask, request, json
+from PIL import Image, ImageDraw
+
+
+app = Flask(__name__)
+zoom = 3
+
+
+@app.route("/", methods=['GET', 'POST'])
+def draw_scheme():
+    state_matrix = json.request
+    state_dictionary = {'Not_Available': 'red',
```

Comment:
you can place state dict outside of any functions, it's constant so you'll don't need to pass whis as parameter